_plugins/docs.rb

```diff
@@ -99,20 +100,10 @@ def form(form_name)
   def scenario(scenario_name, endpoint)
     method = endpoint&.fetch('methods')&.first
 
-    scenario_select_by_name_endpoint_method(scenario_name, endpoint, method) ||
-      scenario_select_by_endpoint_method(endpoint, method) ||
+    scenario_select_by_endpoint_method(endpoint, method) ||
       scenario_select_by_name(scenario_name)
   end
 
-  def scenario_select_by_name_endpoint_method(name, endpoint, method)
-    return unless name && endpoint && method
-    @spec['scenarios'].find do |r|
-      r['name'] == name &&
-        r['endpoint'] == endpoint['path'] &&
-        r['method'] == method
-    end
-  end
-
```

Comment:
gonna take me a while to wrap my head around the removal of this.

Reply:
thanks for walking me through this. Looks good to me, though some of the file paths are a bit much, but I guess they need to be unique so jekyll does not freak out and serve the wrong stuff.